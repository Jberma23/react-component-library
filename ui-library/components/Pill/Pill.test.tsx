import { render } from "@testing-library/react";
import { Pill } from ".";

describe(Pill, () => {
  it("renders with children and defaults", () => {
    const { asFragment } = render(<Pill>Hot Dogs</Pill>);
    expect(asFragment()).toMatchSnapshot();
  });

  it("passes all a11y checks", () => {
    expect(<Pill>Kielbasa</Pill>).toBeA11yWhenRendered();
  });
});
