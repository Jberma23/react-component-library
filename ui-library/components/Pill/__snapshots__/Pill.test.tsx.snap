// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pill renders with children and defaults 1`] = `
<DocumentFragment>
  <div
    class="leading-body font-semibold text-body-regular w-fit px-[0.75rem] py-[0.125rem] border-1 rounded-[50px] whitespace-nowrap bg-primary-50 border-gray-700 text-gray-800"
  >
    Hot Dogs
  </div>
</DocumentFragment>
`;
